test/phpunit/PhysicalobjectCsvImporterTest.php

```diff
@@ -59,17 +59,17 @@ public function setUp(): void
                 "\n",
                 $this->csvData + $this->csvData
             ),
-            'root.csv' => $this->csvData[0],
+            'unreadable.csv' => $this->csvData[0],
             'error.log' => '',
         ];
 
         // setup and cache the virtual file system
         $this->vfs = vfsStream::setup('root', null, $directory);
 
         // Make 'root.csv' owned and readable only by root user
-        $file = $this->vfs->getChild('root/root.csv');
+        $file = $this->vfs->getChild('root/unreadable.csv');
         $file->chmod('0400');
-        $file->chown(vfsStream::OWNER_ROOT);
+        $file->chown(vfsStream::OWNER_USER_1);
```

Comment:
What if the tests are run by user 1?

Reply:
Yeah, the test will fail if run by vfsStream::OWNER_USER_1 (which refs UID 1).  UID is a reserved system UID though, so I assume there is very little chance that the test will ever be run by that user.   If my assumption is wrong, and UID 1 is used to run the test suite in some cases, then I agree that we need a better solution.

It's not ideal, but a lot easier than "fixing" the Docker "run as root" issue.

Reply:
Right, lets hope that's okay, I agree than fixing the Docker root issue can wait for another time, heheh.